v2/mfa/totp/totp-for-all-users.mdx

```diff
@@ -327,6 +327,18 @@ function App() {
 }
 ```
 
+:::important
+If you are using `useRoutes` or `createBrowserRouter` or have routes defined in a different file, use it in the following way. [Issue Reference](https://github.com/supertokens/supertokens-auth-react/issues/581#issuecomment-1246998493)
+```tsx
+const routes = getSuperTokensRoutesForReactRouterDom(reactRouterDom, [^{recipePreBuiltUINameCapitalLetters}]);
```

Comment:
This doesn't match the recipe list above.

Reply:
Fixed